Use the property_set_xx_oneway interface

more36 opened the pull request:
g/5260

rootcasue: Setting property may block the bluetoothd thread, so change it to the property_set_xx_oneway interface, which does not wait for the setting result to return.

*Note: Please adhere to [Contributing Guidelines](https://github.com/open-vela/docs/blob/dev/CONTRIBUTING.md).*

## Summary

*Update this section with information on why change is necessary,
 what it exactly does and how, if new feature shows up, provide
 references (dependencies, similar problems and solutions), etc.*

## Impact

*Update this section, where applicable, on how change affects users,
 build process, hardware, documentation, security, compatibility, etc.*

## Testing

*Update this section with details on how did you verify the change,
 what Host was used for build (OS, CPU, compiler, ..), what Target was
 used for verification (arch, board:config, ..), etc. Providing build
 and runtime logs from before and after change is highly appreciated.*

more36 committed:
Use the property_set_xx_oneway interface

v/53341

rootcasue: Setting property may block the bluetoothd thread, so change it to the property_set_xx_oneway interface, which does not wait for the setting result to return.

Signed-off-by: duqunbo <duqunbo@xiaomi.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/open-vela/frameworks_bluetooth?pullRequest=38) <br/>All committers have signed the CLA.

more36 committed:
HFP: Adds a function to delete list node resources

v/53343

rootcause: The memory allocated when the node is deleted is not freed because there is no delete function.

Signed-off-by: duqunbo <duqunbo@xiaomi.com>

more36 committed:
Merge branch 'dev' into dev-fork
